databridge.examples/databridge.examples.mqtt-jsonata-aas/pom.xml

```diff
@@ -117,6 +117,14 @@
 				</exclusion>
 			</exclusions>
 		</dependency>
+
+		<!-- https://mvnrepository.com/artifact/org.awaitility/awaitility -->
+		<dependency>
```

Comment:
I don't think this dependency is being used anywhere.

Reply:
removed

Reply:
This is still not updated.
Please update this.